Dockerfile

```diff
@@ -8,7 +8,8 @@ WORKDIR /app
 COPY package*.json ./
 
 # Install the dependencies
-RUN npm install
+RUN npm install \
+      && npm run prepare
```

Comment:
This line should be to line 20 (after all of the configs get copied in)

Reply:
(That silences the warning for me)

Reply:
@AlexAxthelm this is the only required change. Everything else looks good, and I got it running on my machine

Reply:
Are those used in the install process? I thought they were all build-time relevant.

Reply:
No no!
`npm run install` should stay where it is
`npm run prepare` needs to go later

:-)

Reply:
@AlexAxthelm I was thinking like this. Only the preparation script requires the configs (but anyway, doesn't matter so much):
``` dockerfile
# Install the dependencies
RUN npm install

# Copy necessary configurations to the working directory
COPY postcss.config.cjs ./
COPY svelte.config.js ./
COPY tailwind.config.ts ./
COPY tsconfig.json ./
COPY vite.config.ts ./

# Prepare svelte-kit
RUN npm run prepare

```